src/core/internal/parseoptions.d

```diff
@@ -94,7 +94,24 @@ bool parseOptions(CFG)(ref CFG cfg, string opt)
                 static if (!is(typeof(__traits(getMember, cfg, field)) == function))
                 {
                     case field:
-                        if (!parse(name, tail, __traits(getMember, cfg, field), errName))
+                        alias fieldT = typeof(__traits(getMember, cfg, field));
+
+                        static if (is(fieldT == size_t))
+                        {
+                             // names of field what can contain B/K/M suffixes
+                            bool isSuffValue = (
+                                name == "initReserve" ||
+                                name == "minPoolSize" ||
+                                name == "maxPoolSize" ||
+                                name == "incPoolSize"
+                            );
```

Comment:
Can you use an UDA instead of having these special cases creeping in here?

Reply:
You mean mark some `Config` fields by special UDA and then find it?

Reply:
std.traits.hasUDA is not available in druntime

Reply:
You can use `__traits(getAttributes, fieldT)` directly.

Reply:
> You can use __traits(getAttributes, fieldT) directly.

Uhh. Too many code duplication with std.traits.
Anybody knows why it is impossible to use std.traits from druntime?

Reply:
Done